sdk/python/tests/test_historical_retrieval.py

```diff
@@ -597,31 +594,6 @@ def test_historical_features_from_bigquery_sources(
         )
 
 
-@pytest.mark.integration
-def test_timestamp_bound_inference_from_entity_df_using_bigquery():
```

Comment:
What happened to this test?

Reply:
I got rid of this function entirely in today's update, so I'll resolve this comment.